panda/src/glstuff/glShaderContext_src.h

```diff
@@ -40,7 +40,7 @@ class EXPCL_GL CLP(ShaderContext) FINAL : public ShaderContext {
   void reflect_uniform(int i, char *name_buffer, GLsizei name_buflen);
   bool get_sampler_texture_type(int &out, GLenum param_type);
 
-  INLINE bool valid(void);
+  INLINE bool valid(void) OVERRIDE;
```

Comment:
Unfortunately MSVC 2010 does not support placing `override` on inline functions.  This is a compiler bug, but we do compile with MSVC 2010 at the moment.